database/migrations/sqlite/1717749007994908_releases.up.sql

```diff
@@ -4,8 +4,17 @@ CREATE TABLE IF NOT EXISTS releases
     releaseVersion INTEGER, -- release ID given by the client that calls CreateApplicationVersion
     created TIMESTAMP,
     appName VARCHAR,
-    envName VARCHAR,
-    manifest VARCHAR,
+    manifests VARCHAR,
     metadata VARCHAR,
-    PRIMARY KEY(eslVersion, releaseVersion, appName, envName)
+    PRIMARY KEY(eslVersion, releaseVersion, appName)
+);
+
+
+CREATE TABLE IF NOT EXISTS all_releases
```

Comment:
Do we document these tables and their columns anywhere? If not, maybe this is the right place for it. Let's add an explanatory comment to these fields, at least the problematic JSON blobs.

Reply:
I added some comments